### DISCUSSION TITLE
Online FPGA retune problem ( status : unknown )

### DISCUSSION THREAD

**Discussion by kevin861222** [bol-edu/HLS-SOC-Discussions#57] [Q&A] [answered]
以下是在Lab1 實作過程遇到的問題，我先租用了 pynq_05，因為轉移陣地，途中wifi斷線導致（MobaXterm）SSH連線中斷，隨後查看我所租用的 pynq_05 狀態為 unknown
<img width="271" alt="截圖 2023-09-18 下午2 01 30" src="https://github.com/bol-edu/HLS-SOC-Discussions/assets/79128379/c25902f2-bc88-4ef1-a8df-c249c419e3a6">
並且在歸還時顯示

<img width="308" alt="截圖 2023-09-18 下午2 02 19" src="https://github.com/bol-edu/HLS-SOC-Discussions/assets/79128379/cdd80ce7-1a0e-4654-9773-4e2c1f369b78">

請問這樣是否成功歸還？

若未成功歸還，我該如何處理？

**Comment by kevinjantw** [comment, marked as answer]
嗨 你應該是可以重新租借新的pynq
如果有問題的話 請再提問
